pages/documentation.md

```diff
@@ -28,7 +28,7 @@ If you actually want to make the fixes and improvements, then
 you'll need to [fork](https://help.github.com/articles/fork-a-repo) the
 repositories you'd like to work on. Once you've made changes you'd like
 reviewed for integration, submit a [pull
-request](http://help.github.com/send-pull-requests/). However, we recommend
+request](https://help.github.com/send-pull-requests/). However, we recommend
```

Comment:
This change doesn't seem correct. Both the `http` and the `https` version redirect to a third page.